Source/WebCore/html/track/VTTRegion.cpp

```diff
@@ -84,10 +84,8 @@ ExceptionOr<void> VTTRegion::setWidth(double value)
     return { };
 }
 
-ExceptionOr<void> VTTRegion::setLines(int value)
+ExceptionOr<void> VTTRegion::setLines(unsigned value)
```

Comment:
Can we change this function to 'void' only now that we are not throwing exception?